frontend/src/components/sidebar/sidebar.module.scss

```diff
@@ -0,0 +1,12 @@
+.main {
+    background-color: #106cc8;
```

Comment:
colours are likely to be reused and since we're using scss it might make sense to pull it as a variable

Reply:
I agree, I didn't do it this time just because I was told (in the ticket) to not worry about colours yet so I figured it can wait, will change it now